module/gridss.nf

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env nextflow
+
+log.info """\
+------------------------------------
+             G R I D S S 2
+------------------------------------
+Docker Images:
+- docker_image_gridss: ${params.docker_image_gridss}
+"""
+
+include { generate_standard_filename; sanitize_string } from '../external/pipeline-Nextflow-module/modules/common/generate_standardized_filename/main.nf'
+
+process preprocess_BAM_GRIDSS {
+    container params.docker_image_gridss
+
+    publishDir "${params.workflow_output_dir}/intermediate",
+        pattern: "${bam_name}.gridss.working/*",
+        mode: "copy",
+        saveAs: {
```

Comment:
**suggestion (blocking)**: These intermediate files should be organized, currently they're all just being copied right under the `intermediate` directory with no indication of what process they came from or what sample they're grouped with. They should be organized to include the process name and then keep the per-sample directory that GRIDSS already generates